src/DIRAC/Core/Tornado/Server/TornadoREST.py

```diff
@@ -42,28 +48,41 @@ def initializeRequest(self):
           '''
           pass
 
-        # Specify the path arguments
+        # Specify the path arguments, that is, it means that you can add to the request path the value of the option
```

Comment:
Yes, these "path-arguments" are added to the tornado `url` in the `HandlerManager.loadEndpointsHandlers`, which allows to then get them as `args` in `BaseRequestMethod._getMethodArgs` method.

Depending on the implementation of the API, the method may have a path-argument or not, or it may be just numbers, and so on